webapp/src/main.rs

```diff
@@ -103,21 +125,14 @@ async fn main() -> Result<()> {
         .serve(app.into_make_service())
         .await?;
 
+    deletion_task.await??;
```

Comment:
I think this will never be called because the call above blocks the thread and if you hit `ctrl+c` in the console the app will terminate.

Reply:
I think you are right. But it doesn't hurt either does it? :)

Reply:
no, but it's useless 😅